blog/_posts/2017-11-30-bloop-release.md

```diff
@@ -0,0 +1,153 @@
+* Exploring integrations with other build tools like `Maven` and `Gradle`.
+
+### First beta release `0.1`
+
+The first beta version of bloop is out in Maven Central. To add it to your project, head over
+[scalacenter/bloop][].
+
+### Preserving hot compilers
+
+An important feature of `bloop` is its [nailgun][] integration. Nailgun is a client, protocol, and
+server for running Java programs from the command line without incurring the JVM startup overhead.
+With it, `bloop` ensures that you always have a hot compiler in the background to compile your
+project. This brings significant improvements indeed: [a hot compiler can compile up to 18 times
+faster than a hot
+compiler](https://scala-ci.typesafe.com/grafana/dashboard/db/scala-benchmark?var-branch=2.12.x&var-source=vector&var-bench=HotScalacBenchmark.compile&var-host=scalabench@scalabench@).
+
+Keeping a hot compiler in the background is crucial if one takes compilation speed and productivity
+seriously. This is often difficult and takes space of mind. For example, `sbt` throws away a hot
+compiler every time users quit (`Ctrl-C`) and restart, but also when they do `reload` after several
+executions of `compile`. By this rule, the more users modify their sbt build, the less productive
+they are.
+
+Current sbt users can indeed avoid quitting and prefer suspending (`Ctrl-Z`), but they will always
+be punished for modifying their builds. In our experience, changing habits is hard, so we have
+preferred to add a built-in nailgul integration in `bloop` that addresses this problem at its root
+rather than changing people's habits, which is hard and time consuming.
+
+### How does it work?
+
+When `bloop` starts up, it loads a build from a configuration directory passed in by the user, which
+defaults to `.bloop-config`. This directory contains a configuration file per every project, and
+every configuration file contains basic information about the project: name, base directory,
+classpath, source directories, classes directory, Scala and Java options and a few more, which are
+hard to create and maintain.
+
+`bloop` comes with an sbt plugin that allows you to generate these configuration files with the
+execution of a command. When the command is done, you can run your build with `bloop`.
+
+This configuration file is a properties file that we plan on migrating from. In the next weeks, we'd
+like to come up with the design of a configuration file that we can reuse across different tools in
+the Scala ecosystem. Given how common this kind of information is required, we believe that settling
+on an agreed upon configuration format is the way to go.
+
+In the future, we want to explore making `bloop` a server you can ask compilation and test-related
+information from different language servers, presentation compilers and linters.
+
+## Numbers on compilation performance
+
+`bloop` has been created out of the observation that Scala compile times slow down within `sbt` in
```

Comment:
"We created `bloop` because we observed that Scala compile times slow down when within sbt when compared to isolated benchmarks"